packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,152 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+Visual Design Comments:
+
+Brandon's Figma spec has designs for two accordion item heights, 24px and 32px. The 32px will be implemented as this is what we have previously focused on when choosing between the two sizes.
+
+Brandon's Figma spec also only shows designs for one accordion item at a time. How should accordion items in each appearance type be separated from each other (ex. a line between accordion items, transparent space between accordion items, etc.)? Because there's only one accordion item design at a time, it is also unclear if error-visible is a nimble-accordion-item specific state, or if it should affect all accordion items and be used in the nimble-accordion component.
+
+The Figma design does not have a design for the "error" focusVisible state. This might be confusing, as keyboard users who tab to the error accordion will suddenly see the red border and exclamation mark icon disappear in the focusVisible state.
+
+When should each `appearance` type be used? Will there be specific guidance on this, and if so, will it be similar to the guidance for other Nimble components that use the same `appearance` types (ex. the Nimble-Button)?
```

Comment:
A collection of edits and formatting changes to help position this section more clearly:
```suggestion
There are several open questions with the current visual design. They're tracked here for background but won't block initial approval of this component spec.

1. the Figma spec has designs for two accordion item heights, 24px and 32px. The 32px will be implemented first as this matches other components. We can implement the 24px variation when we receive client requests for it.

2. the Figma spec only shows designs for one accordion item at a time. How should accordion items in each appearance type be separated from each other (ex. a line between accordion items, transparent space between accordion items, etc.)? Because there's only one accordion item design at a time, it is also unclear if error-visible is a nimble-accordion-item specific state, or if it should affect all accordion items and be used in the nimble-accordion component.

3. The Figma spec does not have a design for the "error" focusVisible state. This might be confusing, as keyboard users who tab to the error accordion will suddenly see the red border and exclamation mark icon disappear in the focusVisible state.

4. When should each `appearance` type be used? Will there be specific guidance on this, and if so, will it be similar to the guidance for other Nimble components that use the same `appearance` types (ex. the Nimble-Button)?
```

Reply:
Now that I see it in context, I would also be happy with tracking these in a separate visual design issue. That way we don't have to update the spec when we resolve them.

Reply:
Sure, I'll create an issue and link to it in the accordion spec,